### ISSUE TITLE
Closing function call bracket errors are inconsistent 

### ISSUE THREAD

**Issue by kbenzie** [richq/cmake-lint#16]
I recently started using this tool and have noticed an inconsistency in how command bracket errors are produced, for example when a CMake call has a new line before the closing bracket the behaviour is different depending on if the command is indented or not.

This example does not generate a lint error.

```cmake
set(VAR
  foo
  bar
)
```

However if the call is indented, such as within a scope, a lint error is produced.

```cmake
if(${COND})
  set(VAR
    foo 
    bar
  )
endif()
```

```
CMakeLists.txt:2: Mismatching spaces inside () after command [whitespace/mismatch]
```

Personally I dislike the style of having a new line before the last bracket and would like to see the addition of a lint error specifically for this case. Is that something you would be willing to add?

**Comment by richq** [comment]
Yeah, that's a bug. This looks like issue 13 too, so that's at least 2 votes for fixing it now.

**Comment by richq** [comment]
This should be fixed now in v1.3.4 - hopefully @tkruse will be able to update pip in the near future.

**Comment by tkruse** [comment]
should I, or are there other changes pending?

**Comment by richq** [comment]
Please do whenever convenient for you, nothing else pending. Cheers!

**Comment by kbenzie** [comment]
@richq thanks for the update.

**Comment by tkruse** [comment]
1.3.4 released on pypi

**Comment by richq** [comment]
Thanks!
		
        
		
		 ---- On Wed, 24 Feb 2016 19:35:05 +0100  notifications@github.com  wrote ----1.3.4 released on pypi
